charts/prometheus/templates/prometheus-config-configmap.yaml

```diff
@@ -270,8 +270,11 @@ data:
             action: labeldrop
           # Required for multi-namespace mode
           - source_labels: [namespace]
+            {{- if .Values.global.namespaceFreeFormEntry }}
```

Comment:
don't set regex if namespaceFreeFormEntry is set. default regex is then used which just selects everything

Reply:
should there not be something inside the if statment?

Reply:
regex: "^(.*$)"

Reply:
nah i'm just using the default regex when it goes in the if.

Reply:
just added a code comment for clarity around this

Reply:
sorry went through code and didnt see ur comment